src/libaktualizr/package_manager/debianmanager.cc

```diff
@@ -52,6 +52,10 @@ data::InstallationResult DebianManager::install(const Uptane::Target &target) co
   return data::InstallationResult(data::ResultCode::Numeric::kInstallFailed, output);
 }
 
+void DebianManager:: completeInstall() const {
+    LOG_ERROR << "Complete installation should not happen for the debian package manager as its updates are applied immidiatelly";
```

Comment:
This will probably stay dead code because `hasPendingUpdate()` will always be false for the debian case.

It could be a default implementation in PackageManagerInterface which throws an exception ("unimplemented") as it should be a programming error. Then, the debian case will be taken care of by itself.

Reply:
OK. Agree. Just think that assert(0) also could be an option here, but it really depends on an approach to assert usage in the project.